src/tools/rbd/Utils.h

```diff
@@ -110,7 +110,7 @@ int get_formatter(const boost::program_options::variables_map &vm,
 void init_context();
 
 int init(const std::string &pool_name, librados::Rados *rados,
-         librados::IoCtx *io_ctx);
+         librados::IoCtx *io_ctx, bool force = false);
```

Comment:
@dillaman, Thank you for The review! I agree this change should better not be placed in Such a common place like init. I will modify it.